ilp-core/src/main/java/org/interledger/core/InterledgerAddress.java

```diff
@@ -152,6 +153,26 @@ default boolean startsWith(final InterledgerAddressPrefix addressPrefix) {
     return this.startsWith(addressPrefix.getValue());
   }
 
+  /**
+   * Return the Interactions as a list of segments for a given {@link InterledgerAddress}.
+   *
+   * @return List of segments returned as {@code List<String>}.
+   */
+  default List<String> getInteractions() {
+    List<String> interactions = new ArrayList<>();
+    String addressValue = this.getValue();
+
+    if (addressValue.contains("~")) {
+      // @sappenin: Can there be multiple interactions listed as g.a.b~ipr.x.y~ipr1.z.w ?
```

Comment:
> Can there be multiple interactions listed as `g.a.b~ipr.x.y~ipr1.z.w`?

My reading of the [grammar defined in RFC-15](https://github.com/interledger/rfcs/blob/master/0015-ilp-addresses/0015-ilp-addresses.md#address-requirements) says, "yes", although I have a question out to the broader community to confirm. See: https://forum.interledger.org/t/questions-about-interactions-in-ilp-addresses/717

Reply:
I agree with the response in the forum. At this level of the library, I don't think we need to worry about "interactions". Sorry for the false issue in #196.